src/components/about.js

```diff
@@ -0,0 +1,123 @@
+const aboutusinfo4 ="Ever since its rechristening, NIT Patna has been on the fast track of development and has undergone numerous facelifts because of which placement records have witnessed unprecedented growth and is touching new heights as the graph of placement is increasing remarkably.";
+
+const missioninfo1 = "To contribute to India and the World through excellence in scientific and technical education and research; to serve as a valuable resource for industry and society; and to remain a source of pride for all Indians.";
+const missioninfo2 = "To generate new knowledge by engaging in cutting-edge research and to promote academic growth by offering state-of-the-art undergraduate, postgraduate and doctoral programmes";
+const missioninfo3 = "To identify, based on an informed perception of Indian, regional and global needs, areas of specialization upon which the Institute can concentrate";
+const missioninfo4 = "To undertake collaborative projects which offer opportunities for long term interaction with academia and industry";
+const missioninfo5 = "To develop human potential to its fullest extent so that intellectually capable and imaginatively gifted leaders can emerge in a range of profession";
+
+const valuesinfo1 = "Academic integrity and accountability.";
+const valuesinfo2 = "Respect and tolerance for the views of every individual.";
+const valuesinfo3 = "Attention to issues of national relevance as well as of global concern.";
+const valuesinfo4 = "Breadth of understanding, including knowledge of the human sciences.";
+const valuesinfo5 = "Appreciation of intellectual excellence and creativity.";
+const valuesinfo6 = "An unfettered spirit of explorating, rationality and enterprise.";
+
+const campusinfo = "NIT Patna is situated on the South bank of river Ganges behind Gandhi Ghat, one of the most important and sacred places of Patna . The Institute campus is 8 Km from the Patna junction railway station and 20 Km from the Jai Prakash Narayan International Airport, Patna . The entrance to the institute is from Ashok Rajpath and it is about 3Km from the famous Gandhi Maidan. The Institute is residential in nature and provides residential facilities to its students as well as teachers as per availability. Its large campus has a picturesque river view with eyecatching historic buildings presenting a spectacle of architectural and natural beauty.";
+
+const Aboutpage = () => {
+  return (
+    <div className="studentsection">
+      <div className="smainrow">
+        <div className="col-6">
+          <div className="row">
+            <h1>About Us</h1>
+          </div>
+          <div className="row">
+            <h3>{aboutusinfo1}</h3>
+          </div>
+          <div className="row">
+            <h3>{aboutusinfo2}</h3>
+          </div>
+          <div className="row">
+            <h3>{aboutusinfo3}</h3>
+          </div>
+          <div className="row">
+            <h3>{aboutusinfo4}</h3>
+          </div>
```

Comment:
I was wondering if it'd be possible to use `<p>` instead of an h3 inside a div here? Conventionally, all `h` tags are used only for headings, so in my opinion, if we use p here, it would make the code more understandable for newcomers too, as these `h3`s themselves contain entire paragraphs of information here.

Reply:
Sure sir, I have changed it. Previously I used h3 tags as it already had that boldness and size. But, now I have used p tag and styled it the same way with css.